adventure-crew-game/Assets/Scripts/Combat/CombatEntity.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using UnityEngine;
+
+public class CombatEntity : MonoBehaviour
+{
+    public CombatEntity target = null;
+    public Stats stats;
+    public float cooldown = 1.0f;
```

Comment:
this could be calculated as a function of Agility

Reply:
I think you mean the cooldown time for the attack, yes that would make Agility more useful. 
I only used agility when calculating the move speed, it makes much sense if agility is also affecting attack speed